BasedHardware/Omi
# Accelerometer Support #569

#569 

How to Test:

1. Flash the zephyr.uf2 file located in friend/firmware/firmwarev1.0/accel_build.
2. Run the updated app in debug mode and open the console.
3. Wait for the hardware to connect. Then after a while you should see accelerometer data, including the gyroscope appear on the console. 
4.  Throw your friend hard, preferably onto a soft surface like a couch. If you throw hard enough you should get a notification labeled 'ouch.....' . I've tried fake falling and if the parameter fall_number listed in communication.dart is low enough, you can trigger the notification but you may trigger it on accident if you move fast.
